Lombiq.Vsix.Orchard/Exceptions/DependencyToConstructorInjectorException.cs

```diff
@@ -8,19 +8,30 @@ public static class DependencyToConstructorIjectorErrorCodes
         public const string ConstructorNotFound = "ConstructorNotFound";
     }
 
-
     [Serializable]
     public class DependencyToConstructorInjectorException : Exception
     {
-        public string ErrorCode { get; }
+        public string ErrorCode { get; set; }
 
+        protected DependencyToConstructorInjectorException()

```

Comment:
Miért kell a ctoroknak protectednek lenniük?

Reply:
![image](https://user-images.githubusercontent.com/18496537/142758544-2f413866-82c4-4a5b-b9ef-d7c5c5f4f9f5.png)

Reply:
Az első háromra jelen esetben viszont nem értem, miért van szükség egyáltalán.

Reply:
Kimaradt.

Reply:
Ha ezt kitörlöm, ami kimaradt, akkor CA1032 Add the following constructor to DependencyToConstructorInjectorException: public DependencyToConstructorInjectorException() warningot jelez

Reply:
Akkor jelen esetben inkább az analyzert kellene suppresselni.